syntax/index.html

```diff
@@ -493,7 +491,7 @@
         </div>
       </section>
       <section>
-        <h3>Site-wide Location Configuration</h3>
+        <h3>Site-wide Location Configuration and Standard Metadata Filenames</h3>
```

Comment:
Grudgingly ok with this, if everybody does. (We introduce a new notion into the standard that was not the intention, but let us close things.)

Ivan

---
Ivan Herman
Tel:+31 641044153
http://www.ivan-herman.net

(Written on mobile, sorry for brevity and misspellings...)



> On 2 Nov 2015, at 17:56, Jeni Tennison <notifications@github.com> wrote:
> 
> In syntax/index.html:
> 
> > @@ -493,7 +491,7 @@
> >          </div>
> >        </section>
> >        <section>
> > -        <h3>Site-wide Location Configuration</h3>
> > +        <h3>Site-wide Location Configuration and Standard Metadata Filenames</h3>
> I'd be happy to accept a PR with just this change in it.
> 
> —
> Reply to this email directly or view it on GitHub.
>